contracts/test/warp.ts

```diff
@@ -0,0 +1,40 @@
+// (c) 2019-2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import {
+  Contract,
+} from "ethers"
+import { ethers } from "hardhat"
+
+const WARP_ADDRESS = "0x0200000000000000000000000000000000000005";
+let senderAddress =  process.env["SENDER_ADDRESS"];
+// Expected to be a hex string
+let payload = process.env["PAYLOAD"];
+let expectedUnsignedMessage = process.env["EXPECTED_UNSIGNED_MESSAGE"];
+let sourceID = process.env["SOURCE_CHAIN_ID"];
+
+describe("IWarpMessenger", function () {
+  this.timeout("30s")
```

Comment:
might be worth setting up a global mocha config file to specify timeouts. Multiple tests are using the 30 second timeout.